data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -274,6 +274,7 @@
 
 REGISTERED_TIER_DEID_CLEANING_CLASSES = [
     (QRIDtoRID,),  # Should run before any row suppression rules
+    (FillSourceValueTextFields,),
```

Comment:
This should be:
```suggestion
    (StringFieldsSuppression,),
``` 
This is the wrong class.

Reply:
🙈😹😅